source/glest_game/gui/gui.cpp

```diff
@@ -777,8 +795,14 @@ void Gui::computeInfoString(int posDisplay){
 	lastPosDisplay = posDisplay;
 
 	display.setInfoText(computeDefaultInfoString());
+	Config &configKeys = Config::getInstance(std::pair<ConfigType,ConfigType>(cfgMainKeys,cfgUserKeys));
+	string commandKeyName = "CommandKey" + intToStr(posDisplay+1);
 
 	if(posDisplay!=invalidPos && selection.isCommandable()){
+		string hotkey = "";
+		if (posDisplay < 10) { // there's 10 configurable command hotkeys
+			hotkey=lang.getString("HotKey")+": "+SDL_GetKeyName(configKeys.getSDLKey(commandKeyName.c_str())) +"\n\n";
```

Comment:
This commit adds the hotkey text on hover. It's going to need translations for the text "Hotkey".